Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -123,67 +124,86 @@ private static InputActionsEditorWindow GetOrCreateWindow(int id, out bool isAlr
             return GetWindow<InputActionsEditorWindow>();
         }
 
+        private static GUIContent GetEditorTitle(InputActionAsset asset, bool isDirty)
+        {
+            var text = asset.name + " (Input Actions Editor)";
+            if (isDirty)
+                text = "(*) " + text;
+            return new GUIContent(text);
+        }
+
         private void SetAsset(InputActionAsset asset, string actionToSelect = null, string actionMapToSelect = null)
         {
-            m_AssetPath = AssetDatabase.GetAssetPath(asset);
-            var serializedAsset = new SerializedObject(asset);
```

Comment:
ok looks like this moved to TryUpdateFromAsset

Reply:
Correct. Why repeat the same code in multiple places.